src/gtest/test_noteencryption.cpp

```diff
@@ -100,9 +100,10 @@ TEST(noteencryption, api)
 
 uint256 test_prf(
     unsigned char distinguisher,
-    uint256 x,
+    uint252 seed_x,
     uint256 y
 ) {
+    uint256 x = seed_x.inner();
     *x.begin() &= 0x0f;
```

Comment:
Even though this is the test PRF, that would probably be a good change to make because failure of that assertion would indicate a bug in the test.